### PR TITLE
feat: [DHIS2-11207] add stage name and icon in the enrollment event page

### PR DESCRIPTION
**Pull request by simonadomnisoru**

### CODE REVIEW

File: src/core_modules/capture-core/components/Pages/EnrollmentEvent/EnrollmentEventPage.container.js [hunk 1]
```diff
@@ -1,10 +1,34 @@
...
+        shallowEqual,
+    );
     const [mode] = useState(pageMode.VIEW);
-
-    return <EnrollmentEventPageComponent mode={mode} />;
+    const { program } = useProgramInfo(programId);
+    const programStage = [...program.stages?.values()].find(
+        item => item.id === stageId,
+    );
+    const eventAccess = { read: true, write: true };
```
Comment (simonadomnisoru) [inline src/core_modules/capture-core/components/Pages/EnrollmentEvent/EnrollmentEventPage.container.js:26]: The `eventAccess` will change once the edit/view form is implemented [DHIS2-11208](https://jira.dhis2.org/browse/DHIS2-11208)

**Review by huocha — APPROVED** [review]

**Review by huocha — APPROVED** [review]

File: src/core_modules/capture-core/components/WidgetEventEdit/WidgetEventEdit.container.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    classes,
+    programStage,
+    programStage: { name, icon },
+    mode,
+}: Props) => (
+    <div data-test="widget-enrollment">
+        <Widget
+            header={
+                <div className={classes.header}>
+                    <NonBundledDhis2Icon
```
Comment (JoakimSM) [inline src/core_modules/capture-core/components/WidgetEventEdit/WidgetEventEdit.container.js:32]: Maybe left-align the program stage name if there is no icon object at all. See `StageOverview.component.js`.

**Review by JoakimSM — APPROVED** [review]
Looks good @simonadomnisoru 👍 One minor comment.

File: src/core_modules/capture-core/components/WidgetEventEdit/WidgetEventEdit.container.js (hunk 2 above)
Comment (simonadomnisoru) [reply src/core_modules/capture-core/components/WidgetEventEdit/WidgetEventEdit.container.js]: Thanks for the feedback! I added it.

**Pull request merged**